scripts/generate_docs.sh

```diff
@@ -1,5 +1,25 @@
 #!/usr/bin/bash
 
+RELEASE=""
+case "$1" in
+  -i|--increment)
+        CURRENT_RELEASE=$(git tag | tail -n1)
+        RELEASE="${CURRENT_RELEASE%.*}.$((${CURRENT_RELEASE##*.}+1))"
+        ;;
+  -k|--keep) # don't change anything, just keep current release
+        ;;
+  -r|--release) RELEASE=$2
```

Comment:
Cool. What to add `--version` or `--get-version` to see current version and then stop?

Reply:
added